### DISCUSSION TITLE
Foocus uses too much CPU (%100)

### DISCUSSION THREAD

**Discussion by rcpylmz1966** [lllyasviel/Fooocus#2643] [Q&A] [answered]
**System features:
Ryzen 5 3500x 6 Core-Processor
32 GB Ram, 3600 MHz
Nvidia Asus Dual Geforce Gtx 1660 Super 6GB**

When I use the Foocus system, the CPU is used at 100% and photos are produced extremely slowly. Is it normal to use this much CPU? And it doesn't use any memory from the graphics card. Is there any way I can solve this problem?

**Comment by Rogala** [comment]
My system features:
CPU 12th Gen Intel(R) Core(TM) i3-12100F 4(8)
Ram Kingston Fury DDR4-3600 MHz 32 GB
Video NVIDIA GeForce GTX 1650 SUPER 4GB (Drivers 551.61)
SSD NVMe KINGSTON SNV2S1000G 40 GB (swap file 55 GB)
My system is no better than yours.

CPU is loaded for 2 minutes at 95%, then transfers data to the GPU, which is loaded at 88-97% and consumes 3GB (does not take more) of 4GB until the end of the picture creation.
Total GPU memory in Windows 10 is 0.1 out of 16GB (Ram 32/2)

The CPU is at most 30% utilized after the transfer of information.
Fooocus settings when creating a picture by attachment with the final picture itself.

Option to reinstall OS with all drivers.

<details>
<summary>**Settings**</summary>

![1](https://github.com/lllyasviel/Fooocus/assets/7437582/a63f71c9-87c4-4267-aa35-17f921baeb4b)

</details>

**Style []**

<details>
<summary>**Model**</summary>

![2](https://github.com/lllyasviel/Fooocus/assets/7437582/02a144b1-1e89-4e4a-81ea-15a706089d29)

</details>

<details>
<summary>**CLI**</summary>

![3](https://github.com/lllyasviel/Fooocus/assets/7437582/ac2d8a29-1487-4a04-ade1-66033bada3d2)

</details>

<details>
<summary>**Image**</summary>

![2024-03-27_20-00-11_9294](https://github.com/lllyasviel/Fooocus/assets/7437582/3f52842a-9a03-4b74-9fda-0cef7d517882)

</details>

**Reply by rcpylmz1966** [reply]
Maybe that's why I'm using a different model, but I generally don't use the GPU, only the CPU, and frankly, I'm a little worried. I'm a bit suspicious about CPU heating etc. However, it still takes a lot of time to create the photo.

**Reply by Rogala** [reply]
I don't understand why use a CPU when there is a video card that will speed up picture creation several times.
For the sake of interest, I ran it with `--always-cpu` flag and the creation time became about 40 minutes.
Why don't you use a GPU?

**Reply by rcpylmz1966** [reply]
What settings do I need to make to use GPU?

**Reply by rcpylmz1966** [reply]
While prepearing foocus texts, my CPU value is 100%. I don't know why.

**Reply by Rogala** [reply]
No special settings and do not need the main thing is to install the drivers on the video card.
What's your OS, have you changed the run.bat settings?

**Reply by rcpylmz1966** [reply]
My operating system is Windows 11. I installed it via the Pinokio application. I didn't download from here

**Reply by Rogala** [reply, marked as answer]
[Download it directly and you'll be happy](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#windows)

**Comment by chenxinlong** [comment]
How did you run fooocus and what's the startup args & environment k/v ?

**Reply by rcpylmz1966** [reply]
I installed it via the Pinokio application. I'm running it via the Pinocchio application.
